plugins/plugin-babel/plugin.js

```diff
@@ -43,6 +43,8 @@ module.exports = function plugin(snowpackConfig, options = {}) {
       }
       return {
         '.js': {
+          contents: code,
+          // QUESTION: Do I need this?
```

Comment:
+1 in favor of returning only `contents` and not `code`, and handling it within Snowpack